ignite/cmd/plugin.go

```diff
@@ -180,6 +180,7 @@ func linkPluginHook(rootCmd *cobra.Command, p *plugin.Plugin, hook *plugin.Hook)
 	}
 
 	newExecutedHook := func(hook *plugin.Hook, cmd *cobra.Command, args []string) *plugin.ExecutedHook {
+		hook.ImportFlags(cmd)
```

Comment:
nit: shall we add a nil check somewhere in linkPluginHook about `hook`?

Reply:
I don't know if I got it. But we can create a `validate` method for the hook